### ISSUE TITLE
byond client 514 camera bug

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#29757]
(WEB REPORT BY: tinysaturn REMOTE: 206.221.180.138:7777)
# Revision
16ebae11f40f3539400fd07f8cb13e4caaf428d5

# Description
camera doesn't seem to move in certain scenarios
# Steps to Reproduce
be on byond beta
get cubed via trash compactor and be put in the inventory
alternatively, spectate a borer inside a host
# What you Expected
for the camera to follow correctly
# What Actually Happened
camera is disjointed

**Comment by d3athrow** [comment]
if you admin observe a player in a situation like this and are running 514 you end up with a poorly updating viewport or just a black one. 513 seems unaffected by this.

**Comment by DeityLink** [comment]
Yep, it's a Byond bug, I reported it to Lummox and provided a demo, now we wait. As Pomf said downgrading to 513 is a workaround.

http://www.byond.com/forum/post/2690420

**Comment by Eneocho** [comment]
Happens when inside disposals and inside the cloner too.

**Comment by Exxion** [comment]
Yes, it happens when inside anything.

**Comment by Eneocho** [comment]
also happens as a borer

**Comment by Exxion** [comment]
>Yes, it happens when inside anything.

**Comment by Eneocho** [comment]
I know, but better to be specific despite the obviousness when bugreporting.
also, copypasted from the other issue:

Temporary fix: Restart byond client while inside the mech. Camera will be working normally. Bug will return if you exit and reenter.

**Comment by SonixApache** [comment]
for the record this is fixed in 1558, we're just waiting for lummox to push out the update

**Comment by Exxion** [comment]
Update: 1558 is out, only a month after Lummox pushed the fix to this bug to that branch, but he also made a shit ton of other changes and 1558 and everything since is horribly buggy
